util/validators.ts

```diff
@@ -32,6 +32,22 @@ export const validateApiList = (val: string): string => {
   return val;
 };
 
+export const validateProductionApiList = (val: string): string => {
+  let result: boolean;
+  try {
+    const apis = val.split(',');
+    result = apis.every(api => PRODUCTION_REQUEST_API_LIST.includes(api));
+  } catch {
+    throw new Error('it was unable to process the provided data');
+  }
+
+  if (!result) {
+    throw new Error('invalid apis in list');
+  }
+
+  return val;
+};
+
```

Comment:
because these are custom validators for `Joi` it's not so easy to be DRY, so this will have to do